### ISSUE TITLE
installer kills GRUB; can't boot

### ISSUE THREAD

**Issue by tomByrer** [pop-os/installer#130]
I just d/l & attempted install of latest PopOS.
 
Hardware history: Windows 10, then installed a mulitboot loader as part few Linuxi; Manjaroo, Kubuntu, etc.  The Windows boot loader remained, but was down-prioritized in BIOS.

I successfully got the Live USB to work.  After poking around, I started to install.  I deleted my old Kubuntu partition, split it (I think 50Gb is enough?), chose the root, boot (left as old partition), & swap.  I started install...

### **Bug 1**: Install errors did not be echoed to default GUI.

Fortunately I noticed the small icon to show the CLI output for the installation script, & got a error about needing `256Mb` for the boot partition.

### **Bug 2**: Other distros, namely Kubuntu 17.10 had no problem with 128Mb boot partition, but PopOS demands larger.

OK, good thing I used GParted 20 times before.  Not wanting to move my Windows boot partition which followed the Kubuntu GRUB 128Mb partition, I resized the ext4 partitions & created a new 266Mb boot partition.
Installation seemed to work, but on rebooting I was greeted with a GRUB CLI prompt, not PopOS.

### **Bug 3**: PopOS install script did not config GRUB correctly.

I manually prioritized the PopOS bootloader in BIOS, but still stuck at GRUB.
Since the Win10 bootloader remained, I can at least be here.

---

What am I to do to get PopOS to boot from GRUB CLI please?  Or do I need to reinstall Kubutu to get a working GRUB?

**Comment by tomByrer** [comment]
Others find the partition size too small also:

https://www.reddit.com/r/pop_os/comments/8mzlee/installation_problems_on_macbook_due_to_efi_size/

**Comment by mmstick** [comment]
Pop!_OS demands a larger size because root encryption requires it. When root is encrypted, you must then store your initramfs + kernel in the EFI partition, instead. If you're dual booting, then you are more than likely going to run out of space on the EFI partition mid-install.

**Comment by ispanos** [comment]
To me none of those "bugs" are bugs and this issue should be closed.

Bug 1 and 2 aren't actual bugs since the `/boot/efi` partition needs to be larger, especially if you are dual-booting. 

Bug 3 again has nothing to do with `Pop-os` in my opinion. Either uninstall grub completely and add new `systemd-boot` entries in `/boot/efi/loader/entries`, or uninstall `systemd-boot`and configure grub to recognizes the partition for Pop.

**Comment by tomByrer** [comment]
@ispanos IIRC the only fix action was to totally reformat the HD & manually make the root boot partition large enough to git PopOS's Grub.  But I had successfully installed other GRUB-based Linux distros on the same machine, so clearly PopOS config issue, not GRUB.

But TBH IDC; I gave up using Linux; the effort to install & config does not pay off in increased productivity.

**Comment by ispanos** [comment]
It has nothing to do with grub. Pop OS requires a bigger /boot partition to accommodate the files mentioned by mmstick. Pop offers a recovery partition that adds to the size required for the boot partition, but that's actually a feature no a bug.

As for giving up on Linux, I switched to Linux 2 years ago and now I can't go back to windows.  But I spent a lot of time reading the arch wiki and manpages until I started seeing any returns from my time investment.
